Changes

```diff
@@ -21,6 +21,10 @@ Working version
 - #11691, #11706: use __asm__ instead of asm for strict ISO C conformance
   (Xavier Leroy, report by Gregg Reynolds , review by Sadiq Jaffer)
 
+* #11693: Remove use of C99 Variable Length Arrays (VLAs) and therefore the
+  caml_alloc_N function.
```

Comment:
This description needs to be updated now that caml_alloc_N never made it into OCaml 5 (esp. no breaking change).